### PR TITLE
User story 13 & 14

### PR DESCRIPTION
**Pull request by ggerdsen**
User Story 13, User can Login

As a visitor
When I visit the login path
I see a field to enter my email address and password
When I submit valid information
If I am a regular user, I am redirected to my profile page
If I am a merchant user, I am redirected to my merchant dashboard page
If I am an admin user, I am redirected to my admin dashboard page
And I see a flash message that I am logged in

User Story 14, User cannot log in with bad credentials

As a visitor
When I visit the login page ("/login")
And I submit invalid information
Then I am redirected to the login page
And I see a flash message that tells me that my credentials were incorrect
I am NOT told whether it was my email or password that was incorrect

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -11,18 +11,18 @@ def create
...
         end
       else
-        flash[:error] = "Your password is incorrect, try again."
+        flash[:error] = "Your login credentials are incorrect"
         render :new
       end
     else
-      flash[:error] = "This is not an email we have a record of, please register."
-      redirect_to "/register"
+      flash[:error] = "Your login credentials are incorrect"
```
Comment (ggerdsen) [inline app/controllers/sessions_controller.rb:24]: Email logic

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -11,18 +11,18 @@ def create
...
           redirect_to '/admin'
-        elsif current_employee?
+        elsif current_merchant?
           redirect_to "/merchant"
         else
           redirect_to "/profile"
         end
       else
-        flash[:error] = "Your password is incorrect, try again."
+        flash[:error] = "Your login credentials are incorrect"
```
Comment (ggerdsen) [inline app/controllers/sessions_controller.rb:20]: Password logic

File: spec/features/logins/login_spec.rb [hunk 3]
```diff
@@ -58,27 +57,72 @@
...
+      expect(current_path).to eq("/login")
+      expect(page).to have_content("Your login credentials are incorrect")
 
     end
   end
 end
 
 
-# User Story 13, User can Login
+
```
Comment (ggerdsen) [inline spec/features/logins/login_spec.rb:113]: We can remove this post merge prior to starting a new branch

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (JBelta) [inline app/controllers/sessions_controller.rb:24]: Keep the flash message vague so we don't specify an incorrect email or password. How effective is this as a security measure?

**Pull request merged**